sklearn/model_selection/_split.py

```diff
@@ -84,6 +84,12 @@ def split(self, X, y=None, groups=None):
 
         test : ndarray
             The testing set indices for that split.
+
+        Note
+        ----
+        Multiple calls to the ``split`` method will not return identical
+        training or testing sets if ``random_state`` parameter exists and is
```

Comment:
How about "Randomized CV splitters may return different results for each call of `split`. This can be avoided (and identical results returned for each split) by setting `random_state` to an integer."